util/util.go

```diff
@@ -106,6 +106,11 @@ func Str2Int64Map(str string) map[int64]struct{} {
 
 // GenLogFields generate log fields.
 func GenLogFields(costTime time.Duration, info *ProcessInfo, needTruncateSQL bool) []zap.Field {
+	if !info.RefCountOfStmtCtx.TryIncrease() {
```

Comment:
So other threads can only access `StmtCtx` by this method?

Reply:
I'm not sure. But it seems we only encounter problems here.😅